R/replace_negatives.R

```diff
@@ -0,0 +1,35 @@
+#' Replace negative incidence with imputed data and redistribute
+#' residuals across observations before adjustment date
+#'
+#' @param data data frame
+#' @param measure death or case
+#' @return data frame after replacement
+#'
+#' @export
+#'
+replace_negatives <- function(data, measure) {
+
+  # find observations with negative inc
+  adjustments <- covidData::get_negative_cases(data)
+
+  # replace negative incidents with imputed data for all replated locations
```

Comment:
Minor spelling note: "incidence" :)

```suggestion
  # replace negative incidence with imputed data for all replated locations
```